src/Middleware/CORS/test/testassets/TestDestination/Program.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
You could move them over instead of copying them. There isn't inherent value in the samples.

Reply:
Cool if I delete the test.htm file then too? Means you can't run these locally without the deployers.

Reply:
Could you move that too? Yeah, that should be ok.